api/bertyverifiablecreds.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+
+package berty.account.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "berty.tech/berty/go/pkg/verifiablecredstypes";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+
+service VerificationCodeSender {
+  rpc SendVerificationCode (SendVerificationCode.Request) returns (SendVerificationCode.Reply);
+  rpc ValidateIdentifier (ValidateIdentifier.Request) returns (ValidateIdentifier.Reply);
+}
+
+message VerifiableCredsNonce {
+  bytes timestamp = 1;
+  bytes nonce = 2;
+  string berty_link = 3;
+  string redirect_uri = 4 [(gogoproto.customname) = "RedirectURI"];
+  string state = 5;
+}
+
+message VerifiableCredsContext {
+  bytes timestamp = 1;
+  string berty_link = 2;
+  CodeStrategy code_strategy = 3;
+  string identifier = 4;
+  string code = 5;
+  string redirect_uri = 6 [(gogoproto.customname) = "RedirectURI"];
+  string state = 7;
+}
```

Comment:
The name of these methods does not seem ideal. Perhaps evoke something more along the lines of "request/init a challenge" and "respond to/complete the challenge". No specific idea but it's a suggestion.